master/internal/db/postgres_tasks.go

```diff
@@ -67,9 +67,14 @@ job_id=EXCLUDED.job_id, log_version=EXCLUDED.log_version
 func (db *PgDB) TaskByID(tID model.TaskID) (*model.Task, error) {
 	var t model.Task
 	if err := db.query(`
-SELECT *
-FROM tasks
-WHERE task_id = $1
+SELECT t.task_id,
+CONCAT('TASK_TYPE_',t.task_type) as task_type,
```

Comment:
this prefixing of TASK_TYPE_ we want to make sure we only for at the API layer when we respond to the user.